src/apps/competitions/emails.py

```diff
@@ -0,0 +1,75 @@
+from utils.email import codalab_send_mail
+
+
+def get_organizer_emails(competition):
```

Comment:
should be property on `Competition` mebe?

Reply:
added `all_organizers` as a property on the competition, which returns a list of the creator and all collabs. so just doing some list comprehension on that property instead.
`[user.email for user in comp.all_organizers]` not sure we necessarily need a property for an email list if we already have a list of users?

Reply:
sounds good